rosbag2_transport/CMakeLists.txt

```diff
@@ -149,19 +149,11 @@ function(create_tests_for_rmw_implementation)
       rosbag2_test_common
       yaml_cpp_vendor)
 
-  # disable the following tests for connext
-  # due to slower discovery of nodes
-  set(SKIP_TEST "")
-  if(${rmw_implementation} MATCHES "(.*)connext(.*)")
```

Comment:
we could keep a `(.*)connext(.*)_cpp` filter here, which would skip the test for the old `rmw_connext_cpp` and `rmw_connext_dynamic_cpp`, but not for `rmw_connextdds`.